.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,107 @@
+--- # Heavily inspired by Airflow's Bug report template at https://github.com/apache/airflow/blob/main/.github/ISSUE_TEMPLATE/airflow_providers_bug_report.yml
+name: Bug report
+description: Problems and issues with code
+title: "[BUG]"
+labels: ["kind:bug", "state:needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      # yamllint disable rule:line-length
+      value: "
+        ## Thank you for your interest in OpenLineage!
+        
+        We appreciate the community's efforts to improve OpenLineage.
+        This form will help us collect a bit of information about the problem you encountered.
+        
+        
+        Note, you do not need to create an issue if you have a change ready to submit!
+        You can open a [pull request](https://github.com/OpenLineage/OpenLineage/pulls) immediately instead."
+      # yamllint enable rule:line-length
+  - type: dropdown
+    attributes:
+      label: OpenLineage integration/client
+      description: Part of code-base that the issue report is about (you can choose more than one)
+      multiple: true
+      options:
+        - Airflow integration
+        - Dagster integration
+        - DBT integration
+        - Flink integration
+        - Spark integration
+        - Python client
+        - Java client
+        - SQL parser
+        - Other
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OpenLineage version
+      description: What name and version of OpenLineage package are you using?
```

Comment:
It's a bit misleading joined with example value. Are we expecting to list all OL-related packages with their versions that user uses or just the package one thinks is buggy?

Reply:
I thought this could be a place to provide information about the one OL package that is faulty so that we know what package and version a user has installed e.g. `openlineage-python==1.14.0`, `io.openlineage:openlineage-spark_2.13:3.5` or `main branch`

Is there a scenario where there are multiple OL packages used in the same time? I was thinking there should be only one version and package here and all other packages (Airflow, Spark etc.) should be included in the next field,

Reply:
I modified the description of two fields related to package versions. Is it more clear now?